packages/fuels-accounts/src/provider/retryable_client.rs

```diff
@@ -150,6 +150,14 @@ impl RetryableClient {
             .map(Into::into)
     }
 
+    pub async fn estimate_predicates(&self, mut tx: Transaction) -> RequestResult<Transaction> {
+        self.client
+            .estimate_predicates(&mut tx)
```

Comment:
This doesn't use `RetryableClient::wrap`, because `RetryableClient::wrap` only supports immutable references in `self.client` calls, which is impossible with `self.client.estimate_predicates(&mut tx)`. Changing `RetryableClient` to accept an `FnMut` instead of `Fn` gives errors about `tx` escaping its lifetime via the closure, so for now I opted to just forward the call to the client without applying retry logic.